src/main/scala/org/hammerlab/guacamole/assembly/AssemblyUtils.scala

```diff
@@ -2,18 +2,40 @@ package org.hammerlab.guacamole.assembly
 
 import htsjdk.samtools.CigarOperator
 import org.apache.spark.Logging
+import org.bdgenomics.utils.cli.Args4jBase
 import org.hammerlab.guacamole.alignment.{AffineGapPenaltyAlignment, ReadAlignment}
 import org.hammerlab.guacamole.reads.MappedRead
 import org.hammerlab.guacamole.reference.{ContigSequence, ReferenceGenome}
 import org.hammerlab.guacamole.util.CigarUtils
 import org.hammerlab.guacamole.variants.ReferenceVariant
 import org.hammerlab.guacamole.windowing.SlidingWindow
+import org.kohsuke.args4j.{Option => Args4jOption}
 
 import scala.collection.JavaConversions._
 
 
 object AssemblyUtils extends Logging {
 
+  trait Arguments extends Args4jBase {
```

Comment:
I'm not crazy about nesting classes like this inside of objects; would prefer making this `assembly.Arguments` rather than `assembly.AssemblyUtils.Arguments` but we've disagreed about that a few times recently; middle ground might be e.g. `assembly.AssemblyArgs` but this wfm if you prefer it